bin/envsetup.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker-compose -f docker-compose.dev.yml up -d
```

Comment:
This is not needed as there is already the Makefile: https://github.com/nimblehq/liveman/blob/main/Makefile 🙈